Apps/Admin/Client/Pages/PatientDetailsPage.razor

```diff
@@ -0,0 +1,109 @@
+@page "/patient-details"
+@layout MainLayout
+@attribute [Authorize(Roles = $"{Roles.Admin},{Roles.Reviewer}")]
+@using HealthGateway.Admin.Client.Store.MessageVerification
+@using HealthGateway.Admin.Client.Store.PatientSupport
+@using HealthGateway.Admin.Common.Constants
+@using HealthGateway.Admin.Client.Components.Support
+@inherits Fluxor.Blazor.Web.Components.FluxorComponent
+
+<PageTitle>Health Gateway Admin Patient Details</PageTitle>
+
+<MudLink Href="/support?details">
+    <MudText Typo="Typo.h5"><MudIcon Icon="fas fa-arrow-left" /> Back</MudText>
+</MudLink>
+
+<HgBannerFeedback
+    Severity="Severity.Error"
+    IsEnabled="HasMessagingVerificationsError"
+    TResetAction="MessageVerificationActions.ResetStateAction"
+    DataTestId="messaging-verification-banner-feedback-error-message">
+    @MessageVerificationState.Value.Error?.Message
+</HgBannerFeedback>
+
+<HgBannerFeedback
+    Severity="Severity.Error"
+    IsEnabled="HasPatientsError"
+    TResetAction="PatientSupportActions.ResetStateAction"
+    DataTestId="user-banner-feedback-error-message">
+    @PatientSupportState.Value.Error?.Message
+</HgBannerFeedback>
+
+<HgBannerFeedback
+    Severity="@Severity.Info"
+    IsEnabled="HasPatientsWarning"
+    TResetAction="PatientSupportActions.ResetStateAction"
+    Class="mt-3"
+    DataTestId="user-banner-feedback-warning-message">
+    <ul>
+        @foreach (string warning in PatientSupportState.Value.WarningMessages)
+        {
+            <li>@warning</li>
+        }
+    </ul>
+</HgBannerFeedback>
+
+<HgBannerFeedback
+    Severity="@Severity.Info"
+    IsEnabled="@(Warning != null)"
+    TResetAction="PatientSupportActions.ResetStateAction"
+    Class="mt-3"
+    DataTestId="user-banner-feedback-status-warning-message">
+    @Warning
```

Comment:
Nit: Could consider renaming this to StatusWarning.

Reply:
Done